lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -665,6 +668,7 @@ public Task updateTask(Task task)
 
```

Comment:
The LOGGER statement on line 634 
should only be called if debug is enabled (to invoke listToString() only when required...